packages/graphql-toolbox/src/modules/HelpDrawer/Resources.tsx

```diff
@@ -81,7 +81,12 @@ const ResourceLinksBlock = ({ listBlockTitle, links }: ResourceLinksBlockProps):
                 {links.map((link) => {
                     return (
                         <li key={link.href} className="mt-6 cursor-pointer">
-                            <a className="flex justify-start items-center" href={link.href} target="_blank">
+                            <a
+                                className="flex justify-start items-center"
+                                href={link.href}
+                                target="_blank"
+                                rel="noopener noreferrer"
```

Comment:
`noreferrer` includes `noopener`, so you can just add `noreferrer` (https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer)

```suggestion
                                rel="noreferrer"
```